BioSQL/BioSeqDatabase.py

```diff
@@ -661,20 +659,26 @@ class scrubs returns of bytearrays and of byte strings converting
     mysql-connector-python in release 2.0.0 of the package.
     """
 
+    def bytearray_to_str(self, s):
```

Comment:
I would suggest making this a private method (change the name to have a leading underscore).

Reply:
Also, since you do not need ``self`` (it was originally a function), this can be a static method:
https://docs.python.org/3/library/functions.html#staticmethod

Basically add the ``@staticmethod`` decorator before the ``def`` line, and remove the unused ``self`` argument.